client-go/cmd/apps.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/deis/deis/client-go/controller/client"
+	"github.com/deis/deis/client-go/controller/models/apps"
+	"github.com/deis/deis/client-go/controller/models/config"
+)
+
+// AppCreate creates an app with the application
+func AppCreate(id string, buildpack string, remote string, noRemote bool) error {
```

Comment:
I don't care either way. Is there a standard throughout the deis project. If the function was only one type, i.e. `func Test(first, second string)` I generally went with but if there were more than one distinct type I added a type to everything to avoid confusion. I'm surprised that `gofmt